sdk/cosmos/azure-cosmos-spark_3-0_2-12/pom.xml

```diff
@@ -0,0 +1,338 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Add MIT license text for the pom file too.

Reply:
Fixed